service/src/main/resources/application.yml

```diff
@@ -28,9 +36,6 @@ spring:
           must-revalidate: true
         use-last-modified: false
   liquibase:
-    url: jdbc:postgresql://${DATABASE_HOSTNAME:localhost}:5432/${DATABASE_NAME:cbas_db}
-    user: ${DATABASE_USER:cbas_user}
-    password: ${DATABASE_USER_PASSWORD:cbas_password}
```

Comment:
TOL: Would removal of this affect any config files in cromhelm repo for CBAS? I am not sure of it myself but just want to make sure we don't break CaaA.

Reply:
This is a good point and I'm not sure either. Since I'm not familiar with cromwhelm, is there a way for me to check/test?

Reply:
I think this should be fine as it seems we are defining this [cbas-api.yaml](https://github.com/broadinstitute/cromwhelm/blob/189473ec60d9016c891d931519f32422fe0f495b/cromwell-helm/templates/cbas-api.yaml#L22-L26).